### PR TITLE
Spelling corrections

### PR DESCRIPTION
**Pull request by scradster**
Hi there

This is a great project and I'm happy to contribute!

Please find my spelling corrections in this PR. Mostly minor corrections, but I also rephrased some sentences where I thought they're hard to understand or confusing.

More content-related PRs to follow :)

Keep up the good work, best regards
Jürg

### CODE REVIEW

File: en/V3-Software_Platform_Requirements.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The Linux operating system is one of the most popular in IoT. It has many featur
 | **3.1.1** | Verify that the bootloader does not allow code to be loaded from arbitrary locations. Locations include both storage (SD, USB, etc.) and network locations (over TCP/IP). | ✓ | ✓ | ✓ |
 | **3.1.2** | Verify bootloader configurations are immutable in production releases. | | ✓ | ✓ |
 | **3.1.3** | Verify that communication interfaces such as, USB, UART, and other variants are disabled or adequately protected during every stage of the device's boot process. | | ✓ | ✓ |
-| **3.1.4** | Verify that the authenticity of the first stage bootloader is verified by a trusted component of which the configuration in read-only memory (ROM) cannot be altered. e.g. CPU Based Secure Boot/Trusted Boot | | ✓ | ✓ |
+| **3.1.4** | Verify that the authenticity of the first stage bootloader is verified by a trusted component of which the configuration in read-only memory (ROM) cannot be altered. E.g. CPU Based Secure Boot/Trusted Boot | | ✓ | ✓ |
```
Comment (cbassem) [inline en/V3-Software_Platform_Requirements.md:20]: Suggest to change to  `altered (e.g. CPU Based Secure Boot/Trusted Boot).` to stay in-line with other uses of e.g.

**Review by cbassem — CHANGES_REQUESTED** [review]
Thank you for this thorough review! One comment with a requested change.

File: en/V3-Software_Platform_Requirements.md (hunk 1 above)
Comment (scradster) [reply en/V3-Software_Platform_Requirements.md:20]: That's the spirit :)

Comment (scradster) [reply en/V3-Software_Platform_Requirements.md]: Done in commit 4c1cc79

**Review by cbassem — APPROVED** [review]

**Pull request merged**